dockerfiles/platform-core/Dockerfile

```diff
@@ -39,3 +39,4 @@ RUN dpkg -i *.deb ; \
 USER hpcc
 ENV PATH="/opt/HPCCSystems/bin:${PATH}"
 ENV HPCC_containerized=1
+ENV HPCC_DLLSERVER_PATH=/var/lib/HPCCSystems/queries
```

Comment:
Should there be and <eol> on the end of the file?

Reply:
Not sure what you mean

Reply:
The diff in github looks as if the last line isn't terminated with /n